### PR TITLE
Add Explainer for TAG review

### PR DESCRIPTION
**Pull request by svgeesus**
TAG needs an explainer before reviewing a document.

They want it to be brief so I just covered new features, not errata.

### CODE REVIEW

**Comment by svgeesus** [comment]
You can [preview the explainer](https://github.com/w3c/PNG-spec/blob/explainer/Third_Edition_Explainer.md)

File: Third_Edition_Explainer.md [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+Video and image content is typically finalized
+on some reference display, called the _mastering display_.
+The content will not contain color values
+that **cannot be reproduced** on the mastering display,
+which sets an **upper bound** on the range of colors
+in the image.
+
+For example, content transmitted in the BT.2100 PQ colorspace,
+which has a theoretical peak luminance of 10,000 cd/m²,
+if often mastered on a display
```
Comment (ProgramMax) [inline Third_Edition_Explainer.md:184]: I think you meant "is often mastered", not "if often mastered".

File: Third_Edition_Explainer.md [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+**gamut mapping** (saturation reduction of un-displayable colors)
+and **tone mapping**
+(compression of luminance to the darkest and brightest values supported by a display).
+
+It is more efficient to provide this information
+than to expect the final display system to compute it on the fly,
+especially for an image sequence,
+where analysis of all frames would be required
+to display the first frame,
+to present the entire sequence in a consistent way.
```
Comment (ProgramMax) [inline Third_Edition_Explainer.md:205]: This sentence was difficult for me to parse.

Maybe break it into smaller pieces? Different wording?
"...compute it on the fly. Especially for an image sequence, where analysis of all frames must be completed prior to displaying the first frame."

File: Third_Edition_Explainer.md [hunk 3]
```diff
@@ -0,0 +1,281 @@
...
+and these are _required metatada_
+for HDR10 content.
+
+As with 'mDCv',
+it is more efficient to provide this information
+than to expect the final display system to compute it on the fly,
+especially for an image sequence,
+where analysis of all frames would be required
+to display the first frame,
+to present the entire sequence in a consistent way.
```
Comment (ProgramMax) [inline Third_Edition_Explainer.md:245]: This sentence could also be updated the same way.

**Review by ProgramMax — APPROVED** [review]
Wow. Great work!

I am marking this as 'approved'. But I think I found a typo that you'll want to fix. And I have some suggestions but they aren't required.

**Pull request merged**